backend/test_multitenant.py

```
@@ -0,0 +1,97 @@
+print(f"✅ School: {admin_user.profile.school}")
+print(f"✅ Students returned: {len(list(students)) if hasattr(students, '__iter__') else Student.objects.filter(school=school1).count()}")
+
+# Test 2: StudentViewSet with anonymous user
+print("\n📝 TEST 2: StudentView Set with anonymous user")
+print("-" * 60)
+request = factory.get('/api/v1/students/')
+request.user = AnonymousUser()
+viewset = StudentViewSet()
+viewset.request = request
+queryset = viewset.get_queryset()
+print(f"✅ User: Anonymous")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)